marketplace-service/src/main/java/com/axonivy/market/service/impl/ProductServiceImpl.java

```diff
@@ -351,7 +299,12 @@ private Pageable refinePagination(String language, Pageable pageable) {
       List<Order> orders = new ArrayList<>();
       for (var sort : pageable.getSort()) {
         SortOption sortOption = SortOption.of(sort.getProperty());
-        Order order = createOrder(sortOption, language);
+        Order order;
+        if (SortOption.STANDARD.equals(sortOption)) {
```

Comment:
please move this if else to createOder function or move the logic create into this one.